public/admin/config.yml

```
@@ -0,0 +1,101 @@
+          - til
+          - meta
+      - {
+          name: updatedDate,
+          label: Updated date,
+          widget: datetime,
+          format: "YYYY-MM-DD",
+          date_format: "YYYY-MM-DD",
+          time_format: false,
+          required: false,
+        }
+      - { name: draft, label: Draft, widget: boolean, default: false }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)